### ISSUE TITLE
Yaml keys with hyphens break syntax highlighting

### ISSUE THREAD

**Issue by aeschli** [textmate/yaml.tmbundle#11]
_From @weevilgenius on December 21, 2015 18:48_

![yaml_screenshot](https://cloud.githubusercontent.com/assets/931455/11938450/c765958e-a7cf-11e5-958d-5cbf0ff3ecba.png)

Editing a yaml file which has a hyphen in an unquoted key name seems to break syntax highlighting for the value of that key

Tested in 0.10.5

_Copied from original issue: Microsoft/vscode#1550_

**Comment by aeschli** [comment]
Reproduced. Snippet
```
----
test1  : dsd
test2  : abc-def
test-3  : abcdef
test-4  : abc-def
```

**Comment by infininight** [comment]
This has been corrected in 953d955b8247a784ed18bb27f5236408dfae0c1b, thanks from the report.
